examples/redis/Examples.StackExchangeRedis/Program.cs

```diff
@@ -0,0 +1,100 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using StackExchange.Redis;
+
+namespace GettingStartedPrometheusGrafana;
+
+public class Program
+{
+    /*
+     * Setup redis service inside local docker.
+     * docker run --name opentelemetry-redis-test -d -p 6379:6379 redis
+     *
+     * If you face any issue with the first command, do the following ones:
```

Comment:
@utpilla could you try this and see if the instructions are still good? I think I added this additional steps when I faced some issues... but dont know if they are relevant to be mentioned here..)

Reply:
I'm not sure how to run into those specific "issues" to try the second set of instructions. The `docker run` command worked for me.

Reply:
if the docker run itself worked, lets removed the rest of the command.. maybe that was something wrong in my machine.